dist/obs-server.spec

```diff
@@ -483,7 +483,7 @@ for i in production.rb ; do
 done
 
 if [ ! -e %{secret_key_file} ]; then
-  ( umask 0077; dd if=/dev/urandom bs=256 count=1 2>/dev/null |sha256sum| cut -d\  -f 1 > %{secret_key_file} )
+  ( umask 0077; RAILS_ENV=production bundle exec rake secret > %{secret_key_file} )
```

Comment:
rake might not be using ruby 2.3 by default, so this might break on systems with a default ruby != 2.3